lib/puppet/indirector/facts/satellite.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-require 'puppet/util/satellite'
+require File.dirname(__FILE__) + '/../../util/satellite.rb'
```

Comment:
`__dir__` is equivalent to `File.dirname(__FILE__)` and shorter. I'd recommend using that instead.

Reply:
I tested this locally, works fine.